8vim/src/main/java/inc/flide/vim8/ui/LauncherActivity.java

```diff
@@ -1,23 +1,33 @@
 package inc.flide.vim8.ui;
 
+import android.annotation.SuppressLint;
```

Comment:
You should not use SuppressLint, if there is an issue with lint, try to find the root cause.

Reply:
yeah understood bro